src/SFA.DAS.AssessorService.Application.Api.UnitTests/Services/StandardServiceTests.cs

```diff
@@ -140,5 +143,27 @@ public async Task When_GettingStandardOptionsByStandardReferenceAndVersion_Then_
             Assert.IsInstanceOf<StandardOptions>(result);
             _mockOuterApiClient.Verify(client => client.Get<StandardDetailResponse>(It.Is<GetStandardByIdRequest>(x => x.Id == getStandardResponse.StandardUId)), Times.Once);
         }
+
+        [Test, AutoData]
+        public async Task When_GettingAllStandardVersions_Then_ReturnsListOfStandards(IEnumerable<Standard> standards)
+        {
+            _mockStandardRepository.Setup(s => s.GetAllStandards()).ReturnsAsync(standards);
+
+            var result = await _standardService.GetAllStandardVersions();
+
+            Assert.IsInstanceOf<IEnumerable<Standard>>(result);
+            Assert.AreEqual(result.Count(), standards.Count());
+        }
+
+        [Test, AutoData]
+        public async Task When_GettingAllStandardVersions_Then_ReturnsListOfStandards(int standardId, IEnumerable<Standard> standards)
```

Comment:
Test name isn't quite right

Reply:
Reworked the test name as it wasn't quite specific enough